lec01.tex

```diff
@@ -211,19 +237,69 @@ \subsection{The Circuit Acceptance Probability Problem (CAPP)}
 \subsection{A Time-Hierarchy Theorem}
 
 \begin{theorem}
-  $\prBPTIME[T \log{T}] \subsetneq \prBPTIME[T]$
+  $\prBPTIME[T] \subsetneq \prBPTIME[\tilde{O}(T)]$
 \end{theorem}
+\begin{proof}
+	We construct a hard promise problem. The input is interpreted as $(<M>,x)$ where $M$ is a machine and $x$ is an input to $M$. The output can be found by simulating the machine $M$ on $x$ for $T$ steps with $T$ random coins and then flipping the output of $M$. 
```

Comment:
I don't understand the definition of the problem. Is the input a machine M, an input x, and a time bound T, and is the output the negation of whether M accepts x in T steps with probability 2/3?

Reply:
I believe that T is not part of the input, but we explicitly set out to design a machine that diagonalizes against T time machines. The output is literally just the negation of M(x), so if x is in the Pi_y (for M), M' outputs no with prob. 2/3, and so on.

(I don't think it's known how to decide if Pr[M(x) = 1] >= 2/3 without enumerating all seeds)